src/Elasticsearch/Endpoints/Ingest/Pipeline/Get.php

```diff
@@ -23,10 +23,9 @@ class Get extends AbstractEndpoint
     public function getURI()
     {
         if (isset($this->id) !== true) {
-            throw new Exceptions\RuntimeException(
```

Comment:
Emmm... yeah that's strange.  I'll inquire :)

Either the test is wrong, or more likely, that's a new "get all pipelines" endpoint (`GET /_ingest`) which isn't documented, and so the REST Spec is wrong :)

Let's leave it alone for now while I poke some people :)